templates/alertmanager.yml.j2

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
Im not sure if it would be practical, but it would make the logic in this file easier to follow if some indentation was used.

Reply:
Unfortunately because of the loops I found that any spacing broke the yaml formatting after the first item.